src/qtt/measurements/acquisition/lock_in_stimulus.py

```diff
@@ -0,0 +1,84 @@
+""" Provides the LockInStimulus"""
+from qilib.configuration_helper import InstrumentAdapterFactory
+from qilib.utils import PythonJsonStructure
+
+
+class LockInStimulus:
+    """ A wrapper that provides methods to control Lock-in of Zurich instrument's UHF-LI devices."""
+    def __init__(self, address: str) -> None:
+        """ Instantiate a LockInStimulus .
+        Args:
+            address: A unique ID of a UHFLI device.
+
+        """
+        self._adapter = InstrumentAdapterFactory.get_instrument_adapter('ZIUHFLIInstrumentAdapter', address)
+        self._uhfli = self._adapter.instrument
+
+    def initialize(self, configuration: PythonJsonStructure) -> None:
+        """ Apply configuration to the UHFLI.
+
+        Args:
+            configuration: The configuration for the UHFLI.
+
+        """
+        self._adapter.apply(configuration)
+
+    def set_demodulation_enabled(self, channel: int, is_enabled: bool) -> None:
+        """ Equivalent to enable data transfer in the demodulators section of the Lock-in tab
```

Comment:
First line of comment should be a single line

Reply:
✔️